### ISSUE TITLE
How to generate multiple samples from a group of key words using "--type sampling"

### ISSUE THREAD

**Issue by etrigger** [dreasysnail/POINTER#16]
For a set of given key words, how to create diverse text generation like the live demo?

**Comment by dreasysnail** [comment]
right. when doing the inference. You run the inference.py with an additional argument "--type sampling"

**Comment by etrigger** [comment]
@dreasysnail @guoyinwang 
In the demo (http://52.247.25.3:8900/), one can generate diverse text based on keywords as shown below.  Right now, the argument '--type sampling' only generate one sentence. How can I create diverse text as shown below? Should I run the inference.py multiple times with different top_k, top_p, and temperature? Will the beam search method mentioned in the paper be released?

Generated from the live demo.
Generated sentence 1 :
he was then announced as the ceo of apple inc ltd . , the founder , and co lead , and co ceo respectively . ,
Generated sentence 2 :
founded by former ceo and co founder of the founders , with the co founder , and current ceo , who currently lead a company .
Generated sentence 3 :
the company was founded by its ceo jeff smith , with former ceo , founder and co owner of facebook and served as the lead ceo .
Generated sentence 4 :
he is the former ceo , former president and the founder , former president and current lead the company .
Generated sentence 5 :
he is the co ceo and co founder of , co founder , and the former chairman and a lead executive of the company .
